src/views/AddItem.jsx

```diff
@@ -3,10 +3,12 @@ import { NavLink } from 'react-router-dom';
 import { addItem } from '../api/firebase';
 
 // sets number of days for each future purchase estimate variable
-const numDaysInEstimate = {
+export const numDaysInEstimate = {
 	soon: '7',
 	kindOfSoon: '14',
 	notSoon: '30',
+	inactive: '60',
+	overdue: '0',
 };
```

Comment:
⚠️ *You don't have to change this in the interest of time, just treat it as a FYI for later*

This is a nit pick, but it would be better if we consolidate `numDaysInEstimate` and `urgencyCategory`. Having 2 sources of truth is not optimal, and they could "drift" and contradict each other if a dev doesn't see them in the same place. 

```javascript
const numDaysInEstimate = {
	soon: '7',
	kindOfSoon: '14',
	notSoon: '30',
	inactive: '60',
	overdue: '0',
}

// you can do some string manipulation to convert camelCase keys -> title case
const urgencyCategory = Object.keys(numDaysInEstimate).map((key) => {label:  key, days: numDaysInEstimate[key]});
```